src/learning-header/LearningHeader.jsx

```diff
@@ -29,10 +32,20 @@ LinkedLogo.propTypes = {
 };
 
 function LearningHeader({
-  courseOrg, courseNumber, courseTitle, intl, showUserDropdown,
+  courseOrg, courseNumber, courseTitle, intl, showUserDropdown, showTabs
 }) {
   const { authenticatedUser } = useContext(AppContext);
+  const [tabs, setTabs] = useState([]);
```

Comment:
Add the "isLoading" setState variable and only render tabs when isLoading is false.

Reply:
Tabs are being render when they are loaded even now.